scripts/intfutil

```diff
@@ -165,6 +166,32 @@ def appl_db_port_status_get(appl_db, intf_name, status_type):
         status = ','.join(new_speed_list)
     return status
 
+def state_db_port_status_get(db, intf_name, field):
+    """
+    Get the port status
+    """
+    full_table_id = PORT_STATE_TABLE_PREFIX + intf_name
+    status = db.get(db.STATE_DB, full_table_id, field)
+    if not status:
+        return "N/A"
+    if field == PORT_SPEED and status != "N/A":
```

Comment:
there is already port_oper_speed_get() to get the speed, so why is this ```if``` check required?

Reply:
Done